sdk/ai/azure-ai-inference/setup.py

```diff
@@ -68,4 +68,7 @@
         "typing-extensions>=4.6.0",
     ],
     python_requires=">=3.8",
+    extras_require={  
+        'trace': ['azure-core-tracing-opentelemetry', 'opentelemetry-sdk', 'azure-monitor-opentelemetry-exporter']  
```

Comment:
we cannot bring otel-sdk or exporter as dependencies - users install them and decide where to export  things on their own. 

```suggestion
        'trace': ['azure-core-tracing-opentelemetry']  
```

Reply:
Okay, but on the other hand i think i have also gotten the impression that there is desire to offer as simple installation and setup procedures as possible. If we do not include them, then i think most users will have to install them separately anyway. But so do you prefer that approach instead of making it as simple as possible for most users (and is there some major downside to installing the additional packages in those cases when they are not used)?

Reply:
It's a fundamental thing in opentelemetry - instrumentation libraries DO NOT use or install SDK or exporters - they provide instrumentations. users may pick another SDK or another exporter. We must not be prescriptive on it.

Adding AzMon exporter is also terrible for MS reputation - we're making Azure Monitor a preferrable destination, but we shouldn't - azure can be observable using 3rd party observability tools.

Reply:
i removed the two additional libraries